src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.IO;
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Callbacks;
+
+namespace Sentry.Unity.Editor.Native
+{
+    public static class BuildPostProcess
+    {
+        [PostProcessBuild(1)]
+        public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
+        {
+            if (target is not (BuildTarget.StandaloneWindows or BuildTarget.StandaloneWindows64))
+            {
+                return;
+            }
+
+            var options = ScriptableSentryUnityOptions.LoadSentryUnityOptions(BuildPipeline.isBuildingPlayer);
+            var logger = options?.DiagnosticLogger ?? new UnityLogger(new SentryUnityOptions());
+
+            try
+            {
+                if (options?.Validate() != true)
```

Comment:
Since we're using pattern matching above, to keep consistent (and since it's preferred over an operator which can be overriden):

```suggestion
                if (options?.Validate() is not true)
```

Reply:
ok